### PR TITLE
tests: add django wrapper testing

### PR DESCRIPTION
**Pull request by razlo**

### CODE REVIEW

File: tests/wrappers/test_django_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    right content.
+    """
+    request_mw = DjangoRequestMiddleware(test_request)
+
+    request_mw.before_request()
+    request_mw.after_request({"content": "bla"})
+
+    runner_mock.assert_called_once()
+    trigger_mock.factory.assert_called_once()
+    assert trace_transport.last_trace.events[
```
Comment (maorlx) [inline tests/wrappers/test_django_wrapper.py:61]: validate method & path equals the ones defined in the test_request fixture

File: tests/wrappers/test_django_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        0].resource['metadata']['Request Data'] == 'test_body'
```
Comment (maorlx) [inline tests/wrappers/test_django_wrapper.py:62]: I would put the test_body into a const, as its being used here twice

File: tests/wrappers/test_django_wrapper.py (hunk 1 above)
Comment (razlo) [reply tests/wrappers/test_django_wrapper.py:61]: we currently don't capture the method :(
added validation for the body

**Pull request merged**